apps/docs/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "@calcom/tsconfig/nextjs.json",
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "noImplicitAny": false
+  }
```

Comment:
In order for the docs to build I enabled "any" types for now. I figured that most of the docs folder are just markdown files anyways.